index.js

```diff
@@ -1,26 +1,23 @@
 module.exports = {
-  'rules': {
-    'brace-style': [2, '1tbs'],
-    'comma-style': [2, 'last'],
-    'indent': [2, 2],
-    'quotes': [2, 'single'],
-    'linebreak-style': [2, 'unix'],
-    'no-alert': 'error',
-    'new-cap': [2, {
-      'newIsCapExceptions': ['baseError']
-    }],
-    'no-underscore-dangle': [0],
-    'no-unneeded-ternary': [0],
-    'no-unused-vars': [2, {
-      'args': 'none'
-    }],
-    'semi': [2, 'always'],
-    'sort-vars': [2, {
-      'ignoreCase': true
-    }],
-    'space-before-blocks': [2, 'always'],
-    'space-before-function-paren': [2, 'never'],
-    'space-in-parens': [2, 'never'],
-    'strict': [0],
+  extends: ['standard', 'standard-react'],
+  env: {
+    mocha: true
```

Comment:
Should we leave this up to the project to add, since it might not always use Mocha? For instance, the design system uses Jest instead.